src/connector/src/source/cdc/external/mysql.rs

```diff
@@ -112,11 +113,55 @@ impl MySqlExternalTable {
             let data_type = mysql_type_to_rw_type(&col.col_type)?;
             // column name in mysql is case-insensitive, convert to lowercase
             let col_name = col.name.to_lowercase();
-            column_descs.push(ColumnDesc::named(
-                col_name.clone(),
-                ColumnId::placeholder(),
-                data_type,
-            ));
+            let column_desc = if let Some(default) = col.default {
+                let snapshot_value = match default {
+                    ColumnDefault::Null => None,
+                    ColumnDefault::Int(val) => match data_type {
+                        DataType::Int16 => Some(ScalarImpl::Int16(val as _)),
+                        DataType::Int32 => Some(ScalarImpl::Int32(val as _)),
+                        DataType::Int64 => Some(ScalarImpl::Int64(val)),
+                        _ => Err(anyhow!("unexpected default value type for integer column"))?,
+                    },
+                    ColumnDefault::Real(val) => match data_type {
+                        DataType::Float32 => Some(ScalarImpl::Float32(F32::from(val as f32))),
+                        DataType::Float64 => Some(ScalarImpl::Float64(val.into())),
+                        DataType::Decimal => Some(ScalarImpl::Decimal(
+                            Decimal::try_from(val).map_err(|err| {
+                                anyhow!("failed to convert default value to decimal").context(err)
+                            })?,
+                        )),
+                        _ => Err(anyhow!("unexpected default value type for float column"))?,
+                    },
+                    ColumnDefault::String(mut val) => {
+                        // append timezone to the default value of timestamptz
+                        if data_type == DataType::Timestamptz {
+                            val.push_str("+00:00");
```

Comment:
This looks a littble bit hacky. Can we use the exact MySQL timestamp format to parse `val` and convert it to timestamptz string in utc. Example:
```rust
    let format = "%Y-%m-%d %H:%M:%S";
    let naive_datetime = NaiveDateTime::parse_from_str(val, format).map_err(...)
    let postgres_timestamptz: DateTime<Utc> = DateTime::<Utc>::from_utc(naive_datetime, Utc);
    let postgres_literal = postgres_timestamptz.format("%Y-%m-%d %H:%M:%S%:z").to_string();
```